Sources/Markdown/Walker/Walkers/MarkupTreeDumper.swift

```diff
@@ -189,6 +189,14 @@ struct MarkupTreeDumper: MarkupWalker {
         dump(heading, customDescription: "level: \(heading.level)")
     }
 
+    mutating func visitOrderedList(_ orderedList: OrderedList) {
```

Comment:
Huh, was this just missing previously?

Reply:
It was using the default visitor before, since it didn't have any associated data to print.